### PR TITLE
Fix name of Goan Konkani, Add Maharashtrian Konkani

### PR DESCRIPTION
**Pull request by Mjbmr**

### CODE REVIEW

**Comment by kartikm** [comment]
Gova -> Goa (Correct spelling :))

**Comment by Mjbmr** [comment]
That's Gova.

Related links:

https://incubator.wikimedia.org/wiki/Wp/gom/%E0%A4%97%E0%A5%8B%E0%A4%B5%E0%A4%BE_%E0%A4%95%E0%A5%8B%E0%A4%82%E0%A4%95%E0%A4%A3%E0%A5%80_%E0%A4%85%E0%A4%95%E0%A4%BE%E0%A4%A6%E0%A5%87%E0%A4%AE%E0%A5%80

http://konkanverter.com/conversion-tool/

https://gerrit.wikimedia.org/r/206083

And google translate's voice.

**Comment by kartikm** [comment]
See: https://incubator.wikimedia.org/wiki/Wp/gom/%E0%A4%97%E0%A5%8B%E0%A4%B5%E0%A4%BE_%E0%A4%95%E0%A5%8B%E0%A4%82%E0%A4%95%E0%A4%A3%E0%A5%80_%E0%A4%85%E0%A4%95%E0%A4%BE%E0%A4%A6%E0%A5%87%E0%A4%AE%E0%A5%80#cite_note-3 - Pronounce is 'Gova', while spelling is 'Goa'

File: data/langdb.yaml [hunk 1]
```diff
@@ -152,9 +152,9 @@ languages:
   gl: [Latn, [EU], galego]
   glk: [Arab, [AS, ME], گیلکی]
   gn: [Latn, [AM], "Avañe'ẽ"]
-  gom: [Deva, [AS], कोंकणी]
-  gom-deva: [gom]
-  gom-latn: [Latn, [AS], Konknni]
+  gom: [gom-deva]
+  gom-deva: [Deva, [AS], गोवा कोंकणी]
+  gom-latn: [Latn, [AS], Gova Konknni]
```
Comment (kartikm) [inline data/langdb.yaml]: Here, it should be Goa.

**Comment by Mjbmr** [comment]
Once again, the name in English is "Goan Konkani" and in Latin Goan Konkani is "Gova Konknni"

**Comment by Mjbmr** [comment]
Use this tool for the conversation: http://konkanverter.com/conversion-tool/

**Comment by Mjbmr** [comment]
As you see also Konkani is written Konknni.

**Pull request closed without merging**